udfs/index.md

```
@@ -230,6 +230,75 @@ function stringify_path(p) {
+Logs a message to the database stdout
+
+##### Syntax
+```javascript
+falkor.log(msg)
+```
+
+##### Parameters
+
+| Parameter | Type | Required | Description |
+|-----------|------|----------|-------------|
+| `message` | string | Yes | message to log |
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)